internal/common/roles/judge.go

```diff
@@ -13,8 +13,8 @@ type EvaluationReturn struct {
 	Evaluations []bool
 }
 type Judge interface {
-	PayPresident() (shared.Resources, bool)
+	PayPresident(presidentSalary shared.Resources) (shared.Resources, bool)
 	InspectHistory(iigoHistory []shared.Accountability) (map[shared.ClientID]EvaluationReturn, bool)
-	DeclareSpeakerPerformance() (result bool, didRole bool)
-	DeclarePresidentPerformance() (result bool, didRole bool)
+	DeclareSpeakerPerformance(inspectBallot bool, conductedRole bool) (result bool, didRole bool)
+	DeclarePresidentPerformance(inspectBallot bool, conductedRole bool) (result bool, didRole bool)
```

Comment:
Are these changes related to the nil pointer issue, or is this about something else?

Reply:
Yeah soz about these, we slightly misaligned the function signatures to their implementations this is a fix for that.

Reply:
That's fine, just edit the PR description to mention this.